httpserver/service_node.cpp

```diff
@@ -197,12 +197,12 @@ ServiceNode::ServiceNode(boost::asio::io_context& ioc,
                          boost::asio::io_context& worker_ioc, uint16_t port,
                          const loki::lokid_key_pair_t& lokid_key_pair,
                          const std::string& db_location,
-                         uint16_t lokid_rpc_port)
+                         LokidClient& lokid_client, uint16_t lokid_rpc_port)
```

Comment:
`lokid_rpc_port` is no longer being used in service_node.cpp

Reply:
removed